front-end/src/components/Register.js

```diff
@@ -35,10 +37,10 @@ const Register = () => {
   const handleRegister = (e) => {
     e.preventDefault();
     const { username, password } = e.target.elements;
-    console.log({ username: username.value, password: password.value });
+    const md5Psw = CryptoJS.MD5(password.value).toString(CryptoJS.enc.Hex);
```

Comment:
Just curious if we need the `CryptoJS.enc.Hex` as an argument for `toString` or if we can just leave the arguments blank. I tried it out locally with both ways and it seemed to be the same result

Reply:
I think it's good to leave in.

Reply:
Gotcha, is that to make it explicit? Cuz when I did it without that argument it was still in hex. Totally cool with having it here, now I'm just curious

Reply:
Yea now looking back on it, it does automatically convert it to a hex string, but explicitly stating it will help us know that we can use other encoding formats in the future.